TeamCode/src/main/java/org/firstinspires/ftc/teamcode/auton/vision/AprilTagAlignment.java

```diff
@@ -0,0 +1,278 @@
+        robot.<DcMotor>get("rightRear").setMode(DcMotor.RunMode.RUN_USING_ENCODER);
+        robot.<DcMotor>get("leftRear").setMode(DcMotor.RunMode.RUN_USING_ENCODER);
+    }
+
+    public void moveYaw() {
+        if (targetTag.ftcPose.yaw > targetYaw) { // Left side down, moving rightFront.
+            setTargetPosition(robot.<DcMotor>get("rightFront"), degreesToMM(Math.abs(targetTag.ftcPose.yaw)));
+            robot.<DcMotor>get("rightFront").setPower(motorRunPower);
+
+            while (robot.<DcMotor>get("rightFront").getTargetPosition() !=
+                    robot.<DcMotor>get("rightFront").getCurrentPosition()) {
+                showMotorStatus(robot.<DcMotor>get("rightFront"));
+            }
+
+            robot.<DcMotor>get("rightFront").setPower(0);
+
+        } else { // Right side down, moving leftFront.
+            setTargetPosition(robot.<DcMotor>get("leftFront"), degreesToMM(Math.abs(targetTag.ftcPose.yaw)));
+            robot.<DcMotor>get("leftFront").setPower(motorRunPower);
+
+            while (robot.<DcMotor>get("leftFront").getTargetPosition() !=
+                    robot.<DcMotor>get("leftFront").getCurrentPosition()) {
+                showMotorStatus(robot.<DcMotor>get("leftFront"));
+            }
+
+            robot.<DcMotor>get("leftFront").setPower(0);
+        }
+    }
+
+    public void moveBearing() {
+        float xDistance = (float) inchesToMM(targetTag.ftcPose.x);
+
+        // Tag is left of where we want it in relation to the robot (targetBearing). Move robot left.
+        if(xDistance < targetBearing) {
+            leftFTarget = xDistance;
+            rightFTarget = -xDistance;
+            leftBTarget = -xDistance;
+            rightBTarget = xDistance;
+        } else { // Move robot right.
+            leftFTarget = -xDistance;
+            rightFTarget = xDistance;
+            leftBTarget = xDistance;
+            rightBTarget = -xDistance;
+        }
+
+        setTargetPosition(robot.<DcMotor>get("leftFront"), leftFTarget);
+        setTargetPosition(robot.<DcMotor>get("rightFront"), rightFTarget);
+        setTargetPosition(robot.<DcMotor>get("leftRear"), leftBTarget);
+        setTargetPosition(robot.<DcMotor>get("rightRear"), rightBTarget);
+
+        while (robot.<DcMotor>get("leftFront").getTargetPosition() !=
+                robot.<DcMotor>get("leftFront").getCurrentPosition() &&
+                robot.<DcMotor>get("rightFront").getTargetPosition() !=
+                    robot.<DcMotor>get("rightFront").getCurrentPosition() &&
+                robot.<DcMotor>get("leftRear").getTargetPosition() !=
+                    robot.<DcMotor>get("leftRear").getCurrentPosition() &&
+                robot.<DcMotor>get("rightRear").getTargetPosition() !=
+                    robot.<DcMotor>get("rightRear").getCurrentPosition()) {
+
+            showMotorStatus(robot.<DcMotor>get("leftFront")); // Also turns on/off motor power.
```

Comment:
Consider abstracting all these `showMotorStatus`calls into a function, since you use it multiple times and it's large.